pkgs/by-name/gp/gpustat/package.nix

```
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, fontconfig
+, libGL
+, xorg
```

```suggestion
, libX11
, libXcursor
, libXi
, libXrandr
```

xorg packages are passed via `callPackage`